pkg/agent/agent_test.go

```diff
@@ -230,23 +257,31 @@ func TestListReleases(t *testing.T) {
 			namespace: "",
 			listLimit: defaultListLimit,
 			releases: []releaseStub{
-				releaseStub{"wordpress", "default", 1, release.StatusDeployed},
-				releaseStub{"wordpress", "dev", 2, release.StatusDeployed},
+				releaseStub{"wordpress", "default", 1, "1.0.0", release.StatusDeployed},
+				releaseStub{"wordpress", "dev", 2, "1.0.0", release.StatusDeployed},
 			},
 			expectedApps: []proxy.AppOverview{
 				proxy.AppOverview{
 					ReleaseName: "wordpress",
 					Namespace:   "default",
-					Version:     "1",
+					Version:     "1.0.0",
 					Status:      "deployed",
 					Icon:        "https://example.com/icon.png",
+					ChartMetadata: chartv1.Metadata{
+						Version: "1.0.0",
+						Icon:    "https://example.com/icon.png",
+					},
 				},
 				proxy.AppOverview{
 					ReleaseName: "wordpress",
 					Namespace:   "dev",
-					Version:     "2",
+					Version:     "1.0.0",
```

Comment:
Is this change trivially correct? If not, maybe add a note in the PR?

Reply:
The version is irrelevant here but I can change the number so the title still applies :+1: